src/main.jl

```diff
@@ -144,30 +114,36 @@ function set_opt!(username, discriminator, service, value)
     opt[string(service)] = value
     user = username * "_" * discriminator
     path = joinpath(pwd(), "data", "opt", user)
-    write(path, json(opt))
+    return write(path, json(opt))
```

Comment:
I would return `nothing` rather than passing back whatever `write` returns. It's not an expected behavior in the "contract" for the `commander` function.